docs/storage/fleek-sdk.md

```diff
@@ -0,0 +1,210 @@
+---
+date: "1"
+
+---
+The Fleek SDK is a tool that allows you to interact programmatically with two features on the Fleek platform **IPFS Pinning** and **Internet Computer (IC) Storage**. It is an alternative to the Fleek Storage JS (the Storage JS uses S3/MinioDB to interact with Fleek's Storage/IPFS products, while the Fleek SDK doesn't).
+
+---
+
+## Internet Computer (IC) Storage
+
+First, the **IC Storage** tool. Which you can use to deploy asset canisters on the Internet Computer (through Fleek's UI), which you then interact with using this SDK tool to store and fetch files on the Internet Computer. This product and the canister deployment flow can be found in the Fleek App under the **Canisters** tab.
+
+## IPFS Direct Pinning
+
+Secondly,  **IPFS Direct Pinning**. This is an alternative to using Fleek's Storage JS. When you store on Fleek's IPFS Storage via the UI, or the Storage JS, you interact with Fleek's IPFS infrastructure through an S3/MinioDB gateway that is on top of it, providing you with human readable live URLs and file metadata that are not native to IPFS. 
```

Comment:
S3 / MinIO (it's not MinioDB)

Reply:
Maybe we can mention the S3 Gateway is using our IPFS APIs underneath and we're just allowing using these IPFS APIs directly (so it's a shortcut). Main benefit of using these new IPFS APIs directly is there's better regional coverage across globe.